index.html

```diff
@@ -1,269 +1,360 @@
+                    </svg>
+                </p>
+                <ul class="navigation-list navigation-list--is-hiden">
+                    <li class="footer-list"><a class="footer-list__link" href="#">Nutrilite</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Artistry</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Satinique</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Amway Home</a></li>
+                </ul>
+            </div>
+            <div class="col-xs-12 col-md-2 footer-category js-footer-toogle">
+                <p class="footer-category__label">ABOUT AMWAY
+                    <svg class="svg-icon footer-category__open">
+                        <use class="js-list-icon"  xlink:href="#icon-amway-icons-open"></use>
+                    </svg>
+                </p>
+                <ul class="navigation-list navigation-list--is-hiden">
+                    <li class="footer-list"><a class="footer-list__link" href="#">Who we are</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Our heritage</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">How we work</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Contact us</a></li>
+                </ul>
+            </div>
+            <div class="col-xs-12 col-md-2 footer-category js-footer-toogle">
+                <p class="footer-category__label">SECURE SHOPPING
+                    <svg class="svg-icon footer-category__open">
+                        <use class="js-list-icon"  xlink:href="#icon-amway-icons-open"></use>
+                    </svg>
+                </p>
+                <ul class="navigation-list navigation-list--is-hiden">
+                    <li class="footer-list"><a class="footer-list__link" href="#">Privacy Policy</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Terms of use</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Satisfaction guarantee</a></li>
+                    <li class="footer-list"><a class="footer-list__link" href="#">Delivery options</a></li>
+                </ul>
+            </div>
+            <div class="col-xs-12 col-md-4 footer-category js-footer-toogle">
+                <p class="footer-category__label">AMWAY BUSINNESS OPPORTUNITY
+                    <svg class="svg-icon footer-category__open">
+                        <use class="js-list-icon"  xlink:href="#icon-amway-icons-open"></use>
+                    </svg>
+                </p>
+                <ul class="col-xs-12 col-md-5 navigation-list navigation-list--is-hiden">
+                    <li class="footer-list"><a class="footer-list__link" href="#">How it works</a></li>

```

Comment:
I will say that li should be treated as navigation-list__item and link inside as navigation-list__item-link